### PR TITLE
Add some documentation bits.

### PR DESCRIPTION
**Pull request by fisx**
I am open to suggestions re. style.  I just hacked in what would have helped me.  (-:

(That's one commit; the other is just cut & paste from #41.)

### CODE REVIEW

File: hedgehog/src/Hedgehog/Internal/Range.hs [hunk 1]
```diff
@@ -210,7 +211,11 @@ linear x y =
 --   >>> bounds 99 $ linearFrom 0 (-10) 20
 --   (-10,20)
 --
-linearFrom :: Integral a => a -> a -> a -> Range a
+linearFrom :: Integral a
+  => a -- ^ Value to shrink toward (the value produced when the size parameter is 0)
+  -> a -- ^ Lower bound (the bottom of the range when the size parameter is 99)
+  -> a -- ^ Upper bound (the top of the range when the size parameter is 99)
+  -> Range a
```
Comment (moodmosaic) [inline hedgehog/src/Hedgehog/Internal/Range.hs]: 😅 Nitpick: Most (if not all?) comments in this codebase end with a `.`— It'd be nice if we do the same also here?

Comment (fisx) [reply hedgehog/src/Hedgehog/Internal/Range.hs]: sure, i can fix all that (give me a few days).  anything else?

Comment (fisx) [reply hedgehog/src/Hedgehog/Internal/Range.hs]: ah, travis.  i'll look into that, too...

**Comment by fisx** [comment]
Ok, I hope this is a bit better.

I am aware that my style is odd and different from the rest of the documentation.  I'm happy to try harder, please let me know if i should!  (-:

**Comment by fisx** [comment]
travis-timeout.  i'll push an empty commit to try again...

(please let me know if you would like me to squash anything / everything.)

File: hedgehog/src/Hedgehog/Internal/HTraversable.hs [hunk 2]
```diff
@@ -7,5 +7,26 @@ module Hedgehog.Internal.HTraversable (
 
 -- | Higher-order traversable functors.
 --
+-- This is used internally to make symbolic variables concrete given an 'Environment'.
+--
+-- The type checker will ask you to write 'HTraversable' instances for your action types.  If you do
```
Comment (thumphries) [inline hedgehog/src/Hedgehog/Internal/HTraversable.hs]: I think from this line down might be better suited to a tutorial than the documentation.

If you could peel it off for now I would be happy to merge

**Comment by thumphries** [comment]
Thanks!

**Comment by fisx** [comment]
thank you!  (-:

**Pull request merged**